src/Microsoft.ML.TimeSeries/SrCnnEntireAnomalyDetector.cs

```diff
@@ -54,6 +56,50 @@ public enum SrCnnDeseasonalityMode
         /// </summary>
         Median = 2
     }
+    public sealed class SrCnnEntireAnomalyDetectorOptions
+    {
+        [Argument(ArgumentType.Required, HelpText = "The name of the input column.", ShortName = "input",
+            SortOrder = 1, Purpose = SpecialPurpose.ColumnName)]
+        public string InputColumnName;
```

Comment:
Question for @harishsk and @yaeldekel : I believe the `[Argument()]` attribute is only necessary for options that are going to be accessed through the command line or an entrypoint, right? So maybe there's no need to add them in here?

Reply:
@harisk has answered me offline, no need to change anything on this PR about it.

---
In reply to: [441177056](https://github.com/dotnet/machinelearning/pull/5202#discussion_r441177056) [](ancestors = 441177056)